examples/rabbitmq-producer/rabbitmq_producer.bal

```diff
@@ -0,0 +1,54 @@
+import ballerina/io;
+import ballerina/rabbitmq;
+
+public function main() {
+    // Creates a ballerina RabbitMQ connection that allows re-usability if necessary.
+    rabbitmq:Connection connection = new({ host: "localhost", port: 5672 });
+
+    // Creates multiple ballerina RabbitMQ channels.
+    rabbitmq:Channel newChannel1 = new(connection);
+    rabbitmq:Channel newChannel2 = new(connection);
+
+    // Declares the queue, MyQueue1.
+    var queueResult1 = newChannel1->queueDeclare(queueConfig = { queueName: "MyQueue1" });
+    if (queueResult1 is error) {
+        io:println("An error occurred while creating the queue, MyQueue1");
+    }
+
+    // Declares the queue, MyQueue2.
+    var queueResult2 = newChannel2->queueDeclare(queueConfig = { queueName: "MyQueue2" });
+    if (queueResult2 is error) {
+        io:println("An error occurred while creating the queue, MyQueue2");
+    }
+
+    // Publishing messages to an exchange using a routing key.
+    // Publishes the message using newChannel1 and the routing key named MyQueue1.
+    worker w1 {
```

Comment:
Shall we also mention in the example why we need to publish using different workers?

Reply:
Resolved